sdk/php/generated/Client.php

```diff
@@ -146,7 +146,8 @@ public function git(
         ServiceId|Service|null $experimentalServiceHost = null,
         ?string $sshKnownHosts = '',
         SocketId|Socket|null $sshAuthSocket = null,
-    ): GitRepository {
+    ): GitRepository
```

Comment:
No idea why the re-generation changed the formatting? CI is failing because of this

Was generated using: `dagger --source . call sdk php generate export --path .`

/cc @dagger/sdk-php